### PR TITLE
[zh-cn]: update the translation of `CanvasRenderingContext2D.canvas` property

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 title: CanvasRenderingContext2D.canvas
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md:2]:
```suggestion
title: CanvasRenderingContext2D：canvas 属性
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+title: CanvasRenderingContext2D：canvas 属性
 slug: Web/API/CanvasRenderingContext2D/canvas
+l10n:
+  sourceCommit: 1f216a70d94c3901c5767e6108a29daa48edc070
 ---
 
 {{APIRef}}
 
-**`CanvasRenderingContext2D.canvas`** 属性是 [Canvas API](/zh-CN/docs/Web/API/Canvas_API) 的一部分，是对与给定上下文关联的{{domxref("HTMLCanvasElement")}}对象的只读引用。如果没有 {{HTMLElement("canvas")}} 元素与之对应，对象值为{{jsxref("null")}} 。
+[Canvas API](/zh-CN/docs/Web/API/Canvas_API) 的 **`CanvasRenderingContext2D.canvas`** 属性是对与给定上下文关联的 {{domxref("HTMLCanvasElement")}} 对象的只读引用。如果没有 {{HTMLElement("canvas")}} 元素与之对应，对象值为 {{jsxref("null")}}。
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md:10]:
```suggestion
[Canvas API](/zh-CN/docs/Web/API/Canvas_API) 的 **`CanvasRenderingContext2D.canvas`** 属性是对与给定上下文关联的 {{domxref("HTMLCanvasElement")}} 对象的只读引用。如果没有与之关联的 {{HTMLElement("canvas")}} 元素，对象值为 {{jsxref("null")}}。
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md [hunk 3]
```diff
@@ -21,15 +21,15 @@ ctx.canvas;
 <canvas id="canvas"></canvas>
 ```
 
-你可以通过 CanvasRenderingContext2D 调用内部的 canvas 属性，获取 canvas 的一个反向引用：
+你可以通过 `CanvasRenderingContext2D` 调用内部的 `canvas` 属性，获取 `canvas` 的一个引用：
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/canvas/index.md:24]:
```suggestion
你可以通过 `CanvasRenderingContext2D` 调用内部的 `canvas` 属性，获取 canvas 元素的引用：
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]

**Pull request merged**